samples/excel/30-range/conditional-formatting-basic.yaml

```
@@ -127,6 +128,20 @@ script:
             });
         }
 
+        async function applyTopBottomFormat() {
+            await Excel.run(async context => {
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const range = sheet.getRange("B21:E23");
+                const conditionalFormat = range.conditionalFormats.add(
+                Excel.ConditionalFormatType.topBottom
```

nit: can we reformat this statement? e.g. move .add to new line
```suggestion
                const conditionalFormat = range.conditionalFormats
                    .add(Excel.ConditionalFormatType.topBottom);
```